alchemist/alchemist-cognitive-agents/src/main/kotlin/it/unibo/alchemist/model/implementations/actions/AvoidFlowField.kt

```diff
@@ -32,5 +36,15 @@ open class AvoidFlowField(
         this.map { it to layer.getValue(it).toDouble() }
             .filter { it.second < currentConcentration }
             .minBy { it.second }?.first ?: currentPosition
+    },
+    TargetSelectionStrategy {
+        with(env.getLayer(targetMolecule).get() as BidimensionalGaussianLayer) {
+            val p = env.getPosition(pedestrian)
+            if (p == null) {
+                env.origin()
+            } else {
+                p + (p - Euclidean2DPosition(centerX, centerY))
```

Comment:
This breaks simulations using non-euclidean positions. Use env.makePosition instead

Reply:
Changed to env.makePosition